cinema/views.py

```diff
@@ -86,7 +86,7 @@ def upload_image(self, request, pk=None):
         item = self.get_object()
         serializer = self.get_serializer(item, data=request.data)
 
-        if serializer.is_valid():
+        if serializer.is_valid(raise_exception=True):
```

Comment:
if you use serializer.is_valid(raise_exception=True) you don't need `if` and response with 400 status